app/directives/forms/edit/reservation.html

```diff
@@ -155,6 +155,23 @@ <h4>Options</h4></div>
                             </div>
                         </div>
                     </div>
+                    <div class="col-xs-4">
+                        <div class="form-group" style="margin-left:10px;margin-top:15px;">
+                            <div class="togglebutton">
+                                <label>
+                                    <input type="checkbox" ng-model="doc.video" ng-false-value="false" ng-false-value="true" ng-change="setVideoUrl(doc)">
```

Comment:
twice `ng-false-value` I think it an error

Reply:
is an error!  Merci will update.